src/pages/workspace/accounting/qbo/advanced/QuickbooksAutoSyncPage.tsx

```diff
@@ -42,7 +42,7 @@ function QuickbooksAutoSyncPage({policy, route}: WithPolicyConnectionsProps) {
             >
                 <HeaderWithBackButton
                     title={translate('common.settings')}
-                    onBackButtonPress={() => Navigation.goBack(ROUTES.WORKSPACE_ACCOUNTING_QUICKBOOKS_ONLINE_ADVANCED.getRoute(policyID))}
+                    onBackButtonPress={() => Navigation.goBack()}
```

Comment:
This will cause the bug when refreshing the page. How about implementing `backTo` approach here?

https://github.com/user-attachments/assets/cef7d516-c5de-4464-a446-40611e57be92

Reply:
@nyomanjyotisa Can you update the testcase to include this case as well? ty